src/hooks/useFetchItems.ts

```diff
@@ -897,3 +899,51 @@ export const useGetProgramsSectionsWithItems = (
 
     });
 };
+
```

Comment:
These `useQuery` hooks should be moved to separate files. This file is WAY too long.

Reply:
@bu3alwa could these 2 hooks be extracted to separate files similar to https://github.com/jellyfin/jellyfin-web/blob/master/src/hooks/useSyncPlayGroups.ts for example?

I think once that is done, this will be good to go. Thanks!